generators/server/templates/gradle/profile_dev.gradle.ejs

```diff
@@ -37,18 +37,14 @@ dependencies {
   <%_ } _%>
   <%_ if (devDatabaseTypeMysql) { _%>
     <%_ if (reactive) { _%>
-    implementation ("dev.miku:r2dbc-mysql") {
-        exclude group: "io.projectreactor", module: "reactor-core"
-        exclude group: "io.projectreactor.addons", module: "reactor-extra"
-        exclude group: "io.projectreactor.netty", module: "reactor-netty"
-    }
+    implementation ("org.mariadb:r2dbc-mariadb")
```

Comment:
There are two different option: maria and mysql: what the point on generating maria when the mysql option is selected?

Reply:
From the Spring Boot 2.7 [release notes](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes#r2dbc-driver-changes):

> r2dbc-mysql, the driver for MySQL, as been removed. Consider using r2dbc-mariadb as a replacement.